API/Services/Tasks/ScannerService.cs

```diff
@@ -430,7 +430,7 @@ private async Task UpdateLibrary(Library library, Dictionary<ParsedSeries, List<
             foreach (var series in librarySeries)
             {
                 await _eventHub.SendMessageAsync(MessageFactory.NotificationProgress, MessageFactory.LibraryScanProgressEvent(library.Name, ProgressEventType.Started, series.Name));
-                await UpdateSeries(series, parsedSeries, allPeople, allTags, allGenres, library.Type);
+                await UpdateSeries(series, parsedSeries, allPeople, allTags, allGenres, library.Type, library);
```

Comment:
If you are altering the signature, don't keep library type and library. Remove the type and get directly from library

Reply:
Fixed with [28bef4b](https://github.com/Kareadita/Kavita/pull/1099/commits/28bef4b4e2a5d1a375456761c8291a7b07f316ec). Resolving comment.